common/mixnet-contract/src/delegation.rs

```diff
@@ -1,40 +1,38 @@
 // due to code generated by JsonSchema
 #![allow(clippy::field_reassign_with_default)]
 
+use crate::HumanAddr;
 use cosmwasm_std::Coin;
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 use std::fmt::Display;
 
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize, JsonSchema)]
 pub struct Delegation {
-    node_identity: String,
+    owner: HumanAddr,
     amount: Coin,
 }
 
 impl Delegation {
-    pub fn new(node_identity: String, amount: Coin) -> Self {
-        Delegation {
-            node_identity,
-            amount,
-        }
+    pub fn new(owner: HumanAddr, amount: Coin) -> Self {
+        Delegation { owner, amount }
     }
 
     pub fn amount(&self) -> &Coin {
         &self.amount
     }
 
-    pub fn node_identity(&self) -> &String {
-        &self.node_identity
+    pub fn owner(&self) -> &HumanAddr {
```

Comment:
Do we want to return `HumanAddr` here, it usually easier if a function returns an owned type

Reply:
I've just looked at the usages of the `owner()` call and you're absolutely right. It would definitely be way easier as in all of the places the call is followed by a `clone()` anyway.